### PR TITLE
Add Sulthan to the list

### PR DESCRIPTION
**Pull request by SulthanNK**
Added my profile data in the public/data/SulthanNK.json file

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/385"></a>

### CODE REVIEW

**Review by adityaraute — APPROVED** [review]

**Review by Nikhil-1503 — APPROVED** [review]

**Review by qwe123coder — APPROVED** [review]
LGTM :rocket: 
thanks for your contrubutions

File: public/data/SulthanNK.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "url": "https://github.com/SulthanNK",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SulthanNK",
+      "icon": "twitter"
+    }
+  ]
+}
```
[the remark this answers is not in the document]
Comment (krishguptadev) [reply public/data/SulthanNK.json:17]: Dang I was requesting changes 🤦 anyways your pr is good but just consider adding a line at the end

[the remark this answers is not in the document]
Comment (qwe123coder) [reply public/data/SulthanNK.json:17]: um its not needed

[the remark this answers is not in the document]
Comment (SulthanNK) [reply public/data/SulthanNK.json:17]: I saw some reviewers are requested to remove the empty line in PRs. So, I didn't put empty lines here. Is it need to change?

[the remark this answers is not in the document]
Comment (qwe123coder) [reply public/data/SulthanNK.json:17]: hey you do not need to add the extra line

[the remark this answers is not in the document]
Comment (eddiejaoude) [reply public/data/SulthanNK.json:17]: The extra line is suggested by GitHub with the red mark, it is good practice 👍

Comment (eddiejaoude) [inline public/data/SulthanNK.json]:
```suggestion
}

```

Comment (eddiejaoude) [reply public/data/SulthanNK.json]: This suggestion should be correct now, then we can merge

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

File: public/data/SulthanNK.json (hunk 1 above)
Comment (qwe123coder) [reply public/data/SulthanNK.json]: hey eddie we need to add extra space ?

[the remark this answers is not in the document]
Comment (krishguptadev) [reply public/data/SulthanNK.json]: imho I think I think we need the last line... Ig the maintainers only ask if there are two empty lines... cc: @Panquesito7

**Comment by SulthanNK** [comment]
Thank you!

**Pull request merged**